search/lib/api/index.js

```diff
@@ -32,6 +32,14 @@ routes.use(provider.routes);
 routes.use(stac.routes);
 routes.use(wfs.routes);
 
+const cloudroutes = express.Router();
+cloudroutes.use('/docs', express.static(path.join(__dirname, '../../docs/index.html')));
+cloudroutes.use('/health', (req, res) => getHealth(res));
```

Comment:
Not entirely sure why this was included to begin with, it's not a STAC endpoint, is it useful for something?

Reply:
I don't think it's very useful but I don't know who is using it.

Reply:
As long as we aren't using it as part of a healthcheck internally it can probably be removed